From 8d69d02b27190c90dc5fd6dcf37420c221ec5660 Mon Sep 17 00:00:00 2001
From: Hongxu Jia <hongxu.jia@windriver.com>
Date: Tue, 24 Jul 2018 17:02:45 +0800
Subject: [PATCH 35/65] set keyboard xlayouts with 'us' by default

Previously only graphic set keyboard xlayouts with 'us' by default,
and text mode did not have the setting and it caused 100% cpu when
loadkey invoked. The hung of loadkey also caused reboot failed.

The setting in default ks works on both of graphic and text.

Revert the following fix:
---------
commit ec53ff6e4a7a06d190925a482371031a2f48509d
Author: Martin Kolman <mkolman@redhat.com>
Date:   Thu Jan 19 18:37:29 2017 +0100

    Use correct default password quality value
---------

Upstream-Status: Inappropriate [oe specific]

Signed-off-by: Hongxu Jia <hongxu.jia@windriver.com>
---
 data/interactive-defaults.ks | 12 +-----------
 1 file changed, 1 insertion(+), 11 deletions(-)

diff --git a/data/interactive-defaults.ks b/data/interactive-defaults.ks
index a906c8c..a2f0555 100644
--- a/data/interactive-defaults.ks
+++ b/data/interactive-defaults.ks
@@ -1,14 +1,4 @@
 # Kickstart defaults file for an interative install.
 # This is not loaded if a kickstart file is provided on the command line.
 firstboot --enable
-
-%anaconda
-# Default password policies
-pwpolicy root --notstrict --minlen=6 --minquality=1 --nochanges --notempty
-pwpolicy user --notstrict --minlen=6 --minquality=1 --nochanges --emptyok
-pwpolicy luks --notstrict --minlen=6 --minquality=1 --nochanges --notempty
-# NOTE: This applies only to *fully* interactive installations, partial kickstart
-#       installations use defaults specified in pyanaconda/pwpolicy.py.
-#       Automated kickstart installs simply ignore the password policy as the policy
-#       only applies to the UI, not for passwords specified in kickstart.
-%end
+keyboard --xlayouts='us'
-- 
2.7.4

